ister.py

```diff
@@ -428,16 +416,36 @@ def copy_os(args, template, target_dir):
         os.chmod("{0}/var/tmp".format(target_dir), stat.S_IRWXU)
         run_command("mount --bind {0}/var/tmp {1}"
                     .format(target_dir, args.statedir))
-    swupd_env = os.environ
-    if template.get("HTTPSProxy"):
-        swupd_env["https_proxy"] = template["HTTPSProxy"]
-        LOG.debug("https_proxy: {}".format(template["HTTPSProxy"]))
 
-    run_command(swupd_command, environ=swupd_env, show_output=True)
+    cmd = "swupd verify --install"
+    cmd += " --path={0}".format(target_dir)
+    cmd += " --manifest={0}".format(template["Version"])
+    if args.contenturl:
+        cmd += " --contenturl={0}".format(args.contenturl)
+    if args.versionurl:
+        cmd += " --versionurl={0}".format(args.versionurl)
+    if args.format:
+        cmd += " --format={0}".format(args.format)
+    cmd += " --statedir={0}".format(args.statedir)
+    if args.cert_file:
+        cmd += " --certpath={0}".format(args.cert_file)
+    if shutil.which("stdbuf"):
+        cmd = "stdbuf -o 0 {0}".format(cmd)
+    cmd_env = get_cmd_env(template)
+    run_command(cmd, environ=cmd_env, show_output=True)
 
     if args.fast_install:
         run_command("rm -rf {0}".format(args.statedir))
 
+def get_cmd_env(template):
```

Comment:
Ah this one slipped by me but it this should have a test function too.

Reply:
Ahh, even this?  Ok  :)